library/src/main/java/com/google/android/exoplayer2/extractor/mp3/Id3Util.java

```diff
@@ -83,211 +90,6 @@ private static boolean canParseMetadata(int majorVersion, int minorVersion, int
         && !(majorVersion == 4 && (flags & 0x0F) != 0);
```

Comment:
If we're supporting other versions, presumably the check for major version here needs removing. Although as per comment elsewhere, we should probably do that separately.